### PR TITLE
Feature/moving parameters generator api

### PR DESCRIPTION
**Pull request by RoyIsland**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,23 +53,26 @@
 - **Always run the simulator after making changes and before merging**, to make sure you didn't break anything.
 - Especially if not sure, **try to get at least 1 person's review before merging**.
 
-## Multiple Jsons generator
+## Multiple jsons generator
```
Comment (bgmoshe) [inline README.md:56]: Actually I thikn JSONs is more correct as JSON is not a word but abbriviation, but it doesn't really matter

Comment (RoyIsland) [reply README.md:56]: I agree, i'll change it back to Jsons

**Comment by benglewis** [comment]
Do you want to create a JSON schema for these JSON files?

**Comment by RoyIsland** [comment]
> Do you want to create a JSON schema for these JSON files?

For the moving parameters json file?

**Pull request merged**